### PR TITLE
update : integration data detail / create function verify & reject klaim rw in detail klaim page

### PR DESCRIPTION
**Pull request by marsellavaleria19**
## Change
- integration data detail klaim RW page.
- create function verify and reject detail klaim rw page.

## Overview
-detail klaim rw page
image

-reject confirmation

https://user-images.githubusercontent.com/44119106/202989910-753a307a-4fe7-47ee-8727-a84a5cf1b481.mp4

-verify confirmation

https://user-images.githubusercontent.com/44119106/202989979-cd0830f9-22fb-4f63-a77d-21afda940be1.mp4

## Evidence
title: Itegrasi data detail klaim rw dan membuat fungsi verifikasi dan tolak klaim rw pada halaman detail klaim RW
project: Jabar Super Apps
participants: @marsellavaleria19 @maulanayuseph @yoslie @agunghide @doohanas 
screenshot: https://user-images.githubusercontent.com/44119106/203206992-c10a662f-c46c-423a-99bf-2103bb4f84e5.png

### CODE REVIEW

File: components/KlaimRW/KlaimRwDetail/index.vue [hunk 1]
```diff
@@ -11,10 +11,10 @@
         </template>
       </BaseButton>
       <div class="flex">
-        <BaseButton class="w-fit border border-red-400 text-red-400 mr-[12px] hover:bg-red-50" @click="isShowRejectConfirmationDialog=true">
+        <BaseButton class="w-fit border border-red-400 text-red-400 mr-[12px] hover:bg-red-50" @click="rejectConfirmationHandle">
```
Comment (yoslie) [inline components/KlaimRW/KlaimRwDetail/index.vue:14]: hello, if data already verified / rejected I think it's better to disable or not showing button reject / verify..
I just realized it so I also change in table list in [here](https://github.com/jabardigitalservice/super-app-cms/pull/22), wdyt?

File: components/KlaimRW/KlaimRwDetail/index.vue [hunk 2]
```diff
@@ -218,6 +218,50 @@ export default {
...
+        this.confirmationDialog.showVerify = false
+        this.informationDialog.showDialog = true
+        this.informationDialog.info = 'Verifikasi akun RW telah berhasil dilakukan.'
+        this.informationDialog.message = 'Email terkait informasi verifikasi telah dikirimkan ke email akun RW bersangkutan.'
+      } catch (error) {
+        this.informationDialog.showDialog = true
+        this.confirmationDialog.showVerify = false
+        this.informationDialog.info = 'Verifikasi akun RW gagal dilakukan.'
+        this.informationDialog.message = ''
+      }
```
Comment (doohanas) [inline components/KlaimRW/KlaimRwDetail/index.vue:260]: ## Ask

Hello @marsellavaleria19, I want to ask, what is the `error` param here for?

**Review by doohanas — APPROVED** [review]

File: components/KlaimRW/KlaimRwDetail/index.vue (hunk 2 above)
Comment (yoslie) [reply components/KlaimRW/KlaimRwDetail/index.vue:260]: hello @doohanas , in my case even it's not used, eslint will detect an error if it's removed.
image
so this error is just act as identifier error condition..

I'm open to any suggestion / solution for this case 🙏

File: components/KlaimRW/KlaimRwDetail/index.vue (hunk 1 above)
Comment (marsellavaleria19) [reply components/KlaimRW/KlaimRwDetail/index.vue:14]: Hello @yoslie.. I agree with you, but I think the design needs to be discussed..

File: components/KlaimRW/KlaimRwDetail/index.vue (hunk 2 above)
Comment (doohanas) [reply components/KlaimRW/KlaimRwDetail/index.vue:260]: How do we remove the parentheses? So, we can write like

```js
try {...}
catch {...}
```

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch 
#cmiiw

Comment (marsellavaleria19) [reply components/KlaimRW/KlaimRwDetail/index.vue:260]: Hello, @doohanas .. the error` param I used to know error from API in catch..

Comment (doohanas) [reply components/KlaimRW/KlaimRwDetail/index.vue:260]: Yes, ofc it's to get the error message from API in the catch. But I don't see the error param used in the catch block. So, I think if not used we can remove the `error` params. Like, my comment above.

**Review by yoslie — APPROVED** [review]

**Review by maulanayuseph — APPROVED** [review]

**Review by agunghide — APPROVED** [review]

File: components/KlaimRW/KlaimRwDetail/index.vue (hunk 2 above)
Comment (marsellavaleria19) [reply components/KlaimRW/KlaimRwDetail/index.vue]: Hello, @doohanas .. Thanks for advice. I've fixed it. This is my commit : https://github.com/jabardigitalservice/super-app-cms/pull/21/commits/9cba78a587a5fc104325532a012867c0c2dd248d

**Pull request merged**